src/Query/Expression.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace ClickHouseDB\Query;
+
+/**
+ * Query expression
```

Comment:
This would deserve more explanation

Reply:
Updated docblock with example and param type.